_posts/2019-07-09-10-praw-uzytecznego-designu.md

```diff
@@ -0,0 +1,124 @@
+
+## 1. Efekt „estetycznej funkcjonalności”
+
+W latach 90 zeszłego stulecia Masaaki Kurosu i Kaori Kashimura postanowili zbadać wpływ atrakcyjności estetycznej na sposób, w jaki postrzega się funkcjonalność. W tym celu poprosili 252 uczestników badania o to, aby ocenili 26 różnych bankomatów, który z nich jest najłatwiejszy w obsłudze, a który najładniejszy. W ten sposób Masaki i Kaori zaobserwowali, że istnieje pewna korelacja - im coś jest piękniejsze tym jest bardziej użyteczne. Dodatkowo piękny wygląd sprawia, że jesteśmy skłonni wybaczyć drobne błędy użyteczności, czar jednak pryska jak bańka mydlana, gdy natrafimy na dużą wadę jak np. brak możliwości złożenia zamówienia.
+
+Spróbujecie przypomnieć sobie sytuację, kiedy musieliście podjąć decyzję i wybrać produkt spośród kilku dostępnych? Jeśli wybraliście coś tylko dlatego, że miało ładną etykietę, to właśnie ulegliście efektowi estetycznej funkcjonalności:)
+
+<div class="img-with-legend">
+<img alt="Strona główna firmy Apple" src="/assets/img/posts/2019-07-09-design/1.jpg" />
+<span class="img-legend">Rys. 1. Strona główna firmy Apple</span>
+</div>
+
+## 2. Prawo Jakoba
+
+Jakob Nielsen, jeden z czołowych specjalistów w dziedzinie użyteczności, a także założyciel Nielsen Norman Group zauważył, że użytkownicy spędzają większość czasu na innych stronach. Oznacza to, że użytkownicy wolą, aby Twoja strona działała tak samo, jak wszystkie inne witryny, które już znają. Dobrym przykładem wykorzystania tej zasady są sklepy sprzedające odzież i obuwie on-line.
+
+Jeśli strona wygląda podobnie do innych, to wtedy użytkownicy będą z łatwością umieli poruszać się po niej, skupiając się na oferowanych produktach.
+
+<div class="img-with-legend">
+<img alt="Wybrany produkt w sklepie on-line marki Kazar" src="/assets/img/posts/2019-07-09-design/2a.jpg" />
+<span class="img-legend">Rys. 2. Wybrany produkt w sklepie on-line marki Kazar</span>
+</div>
+
+<div class="img-with-legend">
+<img alt="Wybrany produkt w sklepie on-line marki Gino Rossi" src="/assets/img/posts/2019-07-09-design/2b.jpg" />
+<span class="img-legend">Rys. 3. Wybrany produkt w sklepie on-line marki Gino Rossi</span>
+</div>
+
+
+## 3. Efekt izolacji
+
+W 1933 roku Hedwig von Restorff odkryła, że przy uczeniu się listy bezsensownych słów łatwiej zapamiętuje się te, które w jakiś sposób wyróżniają się spośród innych, na przykład są napisane większymi literami lub mają inny kolor. Jak się już pewnie domyślacie, ta zasada oprócz tego, że świetnie nadaję się jako pomoc w nauce na jutrzejszą kartkówkę z biologii, to znalazła zastosowanie również w dziedzinie użyteczności. Przyjrzyjmy się chociażby Material Design, czyli jednemu z najpopularniejszych obecnie trendów w tworzeniu interfejsów. Jednym z charakterystycznych elementów tego stylu jest umieszczony w dolnym prawym rogu ekranu okrągły przycisk. Jeśli chcemy zwrócić uwagę użytkownika na dany element strony wyróżnijmy go na tle innych obiektów.
```

Comment:
"umieszczony w dolnym prawym rogu ekranu okrągły przycisk."
 można dodać
 "... , który przesłania inne elementy" lub
 "jest zawieszony ponad innymi elementami"
bo to go wyróżnaia na tle innych lub/oraz to że jest przypięty na stałe

Reply:
myślę że określenie "pływający" wystarczy

Reply:
done